lib/vcs.ml

```diff
@@ -8,8 +8,27 @@ open Bos_setup
 
 (* Version control system repositories *)
 
+module Tag = struct
+  type t = string
+
+  let pp = Fmt.string
+
+  let equal = String.equal
+
+  let to_string x = x
+
+  (* no escaping here in case the user wants to force a literal tag *)
```

Comment:
We should probably make this a doc comment in the `.mli`!

Reply:
I added documentation for the whole module in the `.mli`.